frontend/src/pages/org/browser-profiles-detail.ts

```diff
@@ -362,6 +361,30 @@ export class BrowserProfilesDetail extends TailwindElement {
     </div>`;
   }
 
+  private renderWorkflowName(workflow: ProfileWorkflow) {
+    if (workflow.name)
+      return html`<span class="truncate">${workflow.name}</span>`;
+    if (!workflow.firstSeed)
+      return html`<span class="truncate">${workflow.id}</span>`;
```

Comment:
Might be worth keeping the "(no name)" info and using mono for the id?
```suggestion
      return html`<span class="truncate">${workflow.id}</span>
        <span class="text-neutral-400">${msg("(no name)")}</span>`;
```